### PR TITLE
Event and Assets API cleanup

### PR DESCRIPTION
**Pull request by SteveLasker**
Relatively minor cleanup to make Events easier to grok, minimizing unnecessary details, prior to the Event Centric changes.

### CODE REVIEW

File: content/developers/api-reference/events-api/index.md [hunk 1]
```diff
@@ -471,7 +500,7 @@ You should see the response:
 Event records in DataTrails are tokenized at creation time and referred to in all future API calls by a permanent unique identity of the form:
 
 ```bash
-assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+assets/$ASSET_ID/events/$EVENT_ID
```
Comment (JAG-UK) [inline content/developers/api-reference/events-api/index.md:503]: Interestignly this has been changed in the other direction before becasue people complained it was too hard to spot that $ASSET_ID is actually a variable. 
I don't personally see the problem but that's why it looks like that.
I'll leave it to you whether you want to continue with this change but if you do I recommend you also chnage the xxxx.... in the command block above

File: content/developers/api-reference/events-api/index.md [hunk 2]
```diff
@@ -15,86 +15,113 @@ aliases:
...
+
+Capture the Asset ID by which the events will be associated.
+See [Fetch All Assets](../assets-api/#fetch-all-assets)
+
+```bash
+ASSET_ID=<ASSET_ID>
+```
+
+{{< note >}}
+**Note:** DataTrails will be moving to an event centric design, removing the dependency to reference an Asset.
```
Comment (JAG-UK) [inline content/developers/api-reference/events-api/index.md:44]:
```suggestion
**Note:** DataTrails will soon be moving to an event centric design, removing the need to create and reference Assets to hold your Event collections.
```

**Review by JAG-UK — DISMISSED** [review]
Very small changes, otherwsie great progress

File: content/developers/api-reference/events-api/index.md (hunk 1 above)
Comment (SteveLasker) [reply content/developers/api-reference/events-api/index.md:511]: Good catch. I was being surgical and didn't zoom out to see the consistency issue.
I've made them all consistent, including using CAT to generate the file, with a name that can be directly executed.
If someone just wants the json, they can select the mid section, removing the cat/EOF.

File: content/developers/api-reference/events-api/index.md [hunk 3]
```diff
@@ -15,91 +15,119 @@ aliases:
...
 {{< /note >}}
+
 ## Events API Examples
 
 Create the [bearer_token](/developers/developer-patterns/getting-access-tokens-using-app-registrations) and store in a file in a secure local directory with 0600 permissions.
 
 {{< note >}}
 **Note:** You will need to create an Asset first in order to submit Events against it.
+The dependency on Assets is being deprecated.
+In a future release, Events will be created indecently from Assets.
```
Comment (rcoxhill-01) [inline content/developers/api-reference/events-api/index.md:31]: Was "indecently" the word you meant to use?

File: content/developers/api-reference/events-api/index.md (hunk 1 above)
Comment (rcoxhill-01) [reply content/developers/api-reference/events-api/index.md:511]: Might it not be better to use $ASSET_UUID, so as not to be confused with "Asset ID" (which we use elsewhere to include both the `assets/` and the `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`?

Comment (SteveLasker) [reply content/developers/api-reference/events-api/index.md:511]: This has been confusing at times. I'll change to `ASSET_UUID` when referring to just the UUID.

Comment (SteveLasker) [reply content/developers/api-reference/events-api/index.md]: done

**Review by JAG-UK — APPROVED** [review]
LGTM

**Pull request merged**